engine/Dockerfile

```diff
@@ -41,9 +55,7 @@ RUN pip install -r requirements-enterprise-docker.txt
 
 FROM base AS prod
 
-# This is required for prometheus_client to sync between uwsgi workers
-RUN mkdir -p /tmp/prometheus_django_metrics;
-RUN chown -R 1000:2000 /tmp/prometheus_django_metrics
-ENV prometheus_multiproc_dir "/tmp/prometheus_django_metrics"
+# Change to a non-root user (number is required by Kubernetes runAsNonRoot check)
+USER 1000
```

Comment:
@alexintech actually one thought came to mind. If we move `USER 1000` higher up to line 13, I _think_ we may not need any of the `chown` commands?

Reply:
> If we move USER 1000 higher up to line 13

That will break `RUN apt-get install -y sqlite3 default-mysql-client postgresql-client` [here](https://github.com/grafana/oncall/pull/2053#diff-a25c2322baf692c37d4ecc9c3611772e78dfb8bbaf9cc4ef27b507e60cb3ddc9R49) in `dev` target.

Reply:
that makes sense 👍 

I opened [a PR](https://github.com/grafana/oncall/pull/2132) to move `USER` to the end of the `dev` target so that it is present if a different build `target` is specified to `docker build`